configure.ac

```diff
@@ -33,8 +54,12 @@ AC_PATH_PROG([GROUPADD], [groupadd], [/usr/sbin/groupadd])
 AC_PATH_PROG([CAT], [cat], [/bin/cat])
 AC_PATH_PROG([CHMOD], [chmod], [/bin/chmod])
 AC_PATH_PROG([CHGRP], [chgrp], [/bin/chgrp])
-AC_PROG_SED
-AC_PROG_MKDIR_P
+AC_PROG_CXX
```

Comment:
CXX - braucht ock den einen c++ compiler ?

Reply:
No. autotools complains about missing AC_PROC_CXX because some Makefile.am defines CXX flags. I l look into that, dont think its needed.